app/src/main/java/com/example/admin/personallibrarycatalogue/data/LibraryDatabaseHelper.java

```diff
@@ -0,0 +1,197 @@
+package com.example.admin.personallibrarycatalogue.data;
+
+/**
+ * Created by Mikhail Valuyskiy on 25.05.2015.
+ */
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import com.example.admin.personallibrarycatalogue.data.DatabaseContract.BooksTable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility class which helps work with database
+ */
+public class LibraryDatabaseHelper extends SQLiteOpenHelper {
+
+    // When the database schema was changed, you must increment the database version
+    private static final int DATABASE_VERSION = 2;
+    static final String DATABASE_NAME = "library.db";
+
+    public LibraryDatabaseHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    /**
+     * Creates table to hold book data
+     */
+    @Override
+    public void onCreate(SQLiteDatabase sqLiteDatabase) {
+
+        final String SQL_CREATE_BOOKS_TABLE = "CREATE TABLE " + BooksTable.TABLE_NAME + " (" +
+                BooksTable._ID + " INTEGER PRIMARY KEY AUTOINCREMENT," +
+                BooksTable.AUTHOR + " TEXT NOT NULL, " +
+                BooksTable.TITLE + " TEXT NOT NULL, " +
+                BooksTable.DESCRIPTION + " TEXT, " +
+                BooksTable.COVER + " BLOB );";
+
+        sqLiteDatabase.execSQL(SQL_CREATE_BOOKS_TABLE);
+    }
+
+
+    @Override
+    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int oldVersion, int newVersion) {
+        sqLiteDatabase.execSQL("DROP TABLE IF EXISTS " + DatabaseContract.BooksTable.TABLE_NAME);
+        onCreate(sqLiteDatabase);
+    }
+
+    public int getId(String title, String author) {
+        if ((title != null) && (author != null)) {
+
+            int id = -1;
+            SQLiteDatabase db = this.getWritableDatabase();
+
+            String selectQuery = "SELECT * FROM " + BooksTable.TABLE_NAME +
+                    " WHERE " + BooksTable.TITLE + " =  \"" + title + "\"" +
+                    " AND " + BooksTable.AUTHOR + " =  \"" + author + "\"";
+
+            Cursor cursor = db.rawQuery(selectQuery, null);
+
+            if (cursor.moveToFirst()) {
+                id = (cursor.getInt(cursor.getColumnIndex(BooksTable._ID)));
+            }
+
+            db.close();
+            return id;
+        } else {
+            throw new NullPointerException("Passed title or author is null");
+        }
+    }
+
+    public void deleteBook(int id) {
+        SQLiteDatabase db = this.getWritableDatabase();
+        db.delete(BooksTable.TABLE_NAME, BooksTable._ID + " = " + id, null);
```

Comment:
You can return boolean value to let the caller know if the book really was deleted or not.